pyproject.toml

```diff
@@ -50,5 +47,26 @@ tag-pattern = '^cherry-picker-(?P<version>[vV]?\d+(?:\.\d+){0,2}[^\+]*)(?:\+.*)?
 [tool.hatch.version.raw-options]
 local_scheme = "no-local-version"
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
+fix = true
+
+lint.select = [
```

Comment:
did you want to enable `isort` rules here too? I saw that you removed `PCQA/isort` from pre-commit